src/main/java/com/nextcloud/android/sso/api/Response.java

```diff
@@ -0,0 +1,35 @@
+package com.nextcloud.android.sso.api;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+
+import androidx.annotation.Nullable;
+
+public class Response {
+    private InputStream inputStream;
```

Comment:
maybe we should rename `inputStream` to `body`? (Might be more obvious for using it. In the Nextcloud Android App this is called "response body stream" as well..) https://github.com/nextcloud/android/blob/8955709705a8ae937403b1102a1d11cccc6b9258/src/main/java/com/nextcloud/android/sso/InputStreamBinder.java#L298)